guide/daily-spending-wallet/settings.md

```diff
@@ -0,0 +1,213 @@
+
+During regular use, only lightning network fees will apply to transactions in this wallet. An additional fee may be charged for opening new channels to increase liquidity.
+
+When users send to an on-chain address, a swap fee applies. This fee also applies if the user receives to an on-chain (swap) address. This may cause confusion, as typically the sender is reponsible for paying fees.
+
+</div>
+
+## Privacy
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/settings/privacy.png"
+   retina = "/assets/images/guide/daily-spending-wallet/settings/privacy@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/settings/privacy-big.png"
+   alt-text = ""
+   width = 250
+   height = 541
+   layout = "float-right-desktop -background -shadow"
+   caption = "Privay options focus on the display of sensitive data."
+%}
+
+[Privacy]({{ '/guide/daily-spending-wallet/privacy/' | relative_url }}) settings are straightforward and focused on whether balances are visible or hidden. While there are some best practices for users, much of the responsibility to ensure good privacy rests in the hands of the developers.
+
+</div>
+
+## Security
+
+[Security]({{ '/guide/daily-spending-wallet/security/' | relative_url }}) is all about preventing unwanted access to the application. Basic access options (PIN, FaceID) are configured during [first use]({{ '/guide/daily-spending-wallet/first-use/' | relative_url }}) and users receive regular reminders to review them. Wallet limits are an additional option, only available in settings, to block potentially unwanted high-value payments.
+
+{% include image-gallery.html pages = page.images_security %}
+
+## Wallet backup
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/settings/wallet-backup.png"
+   retina = "/assets/images/guide/daily-spending-wallet/settings/wallet-backup@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/settings/wallet-backup-big.png"
+   alt-text = ""
+   width = 250
+   height = 541
+   layout = "float-right-desktop -background -shadow"
+   caption = "Backup options are typically configured in first use and rarely need to be accessed."
+%}
+
+[First use]({{ '/guide/daily-spending-wallet/first-use/' | relative_url }}) also guides users towards appropriate [backup]({{ '/guide/daily-spending-wallet/backup-and-recovery/' | relative_url }}) options, and users may never need to access this screen. A unique piece of information here is when the last cloud backup was performed. This can give users peace of mind that their information is securely backed up.
```

Comment:
```suggestion
[First use]({{ '/guide/daily-spending-wallet/first-use/' | relative_url }}) also guides users towards appropriate [backup]({{ '/guide/daily-spending-wallet/backup-and-recovery/' | relative_url }}) options, and users may never need to access this screen. A unique piece of information here is when the last cloud backup was performed. This can give users peace of mind that their information is securely being backed up.
```

As it occurs frequently